examples/pub-sub-rewind/react/src/Home.tsx

```diff
@@ -0,0 +1,100 @@
+'use client'
+
+import * as Ably from 'ably';
+import minifaker from 'minifaker';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { useState } from 'react';
+import 'minifaker/locales/en';
+
+export default function Home() {
+  const [alerts, setAlerts] = useState<Array<{id: number, message: string}>>([]);
+  const navigate = useNavigate();
+  const [isLoading, setIsLoading] = useState(false);
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
```

Comment:
there is a more elegant way to do this in React Router: [`useParams()`](https://reactrouter.com/6.30.0/hooks/use-params) or [`useSearchParams()`](https://reactrouter.com/6.30.0/hooks/use-search-params)